python/src/diffusionkit/mlx/mmdit.py

```diff
@@ -209,13 +214,15 @@ def __call__(
         else:
             positional_encodings = None
 
+        timestep_embedding = self.guidance_in(self.t_embedder(timestep))
+
         # MultiModalTransformer layers
         if self.config.depth_multimodal > 0:
             for bidx, block in enumerate(self.multimodal_transformer_blocks):
                 latent_image_embeddings, token_level_text_embeddings = block(
                     latent_image_embeddings,
                     token_level_text_embeddings,
-                    timestep,
+                    timestep_embedding,
```

Comment:
Override original timestep only for dev model

```suggestion
                    timestep,
```